crates/debugger-macro-impl/src/lib.rs

```diff
@@ -0,0 +1,84 @@
+use proc_macro::TokenStream;
+use proc_macro2::TokenStream as TokenStream2;
```

Comment:
Why would we need both?

Reply:
It's a long story - short story: `proc_macro2` has a bunch of nice extensions which make building proc macros much easier. Mostly related to interpolating in/out of other streams via the `quote!` macro. Basically, we want to keep everything in `TokenStream2` until we finally return it from the proc_macro which has to use `TokenStream`.

Reply:
I hope someone just merges the proc_macro2 crate into upstream at some hypothetical future point in time since it's pretty silly. (there are lots of reasons that hasn't happened and may not ever happen... fun rust compiler stuff)

Reply:
Would any of the utilities from [`syn`](https://crates.io/crates/syn) simplify this away?
I used it some when contributing to [`miette`](https://crates.io/crates/miette) a while back and it seemed pretty good at this stuff.

Reply:
🙃